### PR TITLE
Cli branch

### PR DESCRIPTION
**Pull request by LiamMoMurray**
The CLI is CLI'ing for the pre-processing and training models. need to work on the predict / classify section

### CODE REVIEW

**Review by s1m0nWasTaken — COMMENTED** [review]
She's CLI-ing alright pal.. good work keep Tipp-in on.

File: CLI/menus.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+                case 2:
+                    train_menu()
+                case 3:
+                    classify_menu()
+                case 4:
+                    run = False
+                case _:
+                    print("Not a valid choice! Try again. (1,2,3): ")
+        except ValueError:
+            print("WTF")
```
Comment (tadhgryan2012) [inline CLI/menus.py:32]: Please refrain from using profanity

Comment (LiamMoMurray) [reply CLI/menus.py:32]: Your aul one

**Review by s1m0nWasTaken — APPROVED** [review]
lgtm

**Pull request merged**